File: praktikum_algopascal/pascal/biodata_procedure.pas
//prosedur biodata

program biodata;
	uses crt;

procedure boi;
	var
		nama, alamat, tgl: string;
		umur: Integer;
begin
	write('Masukan Nama: ');
	readln(nama);
	write('Masukan Alamat: ');
	readln(alamat);
	write('Masukan Tanggal Lahir: ');
	readln(tgl);
	write('Umur Anda: ');
	readln(umur);
	
	writeln('==============================');
	writeln('Nama Anda: ', nama);
	writeln('Alamat: ', alamat);
	writeln('Tanggal Lahir: ',tgl);
	writeln('Umur: ', umur);
end;

begin
	clrscr;
	boi;
	readln;
end.

File: praktikum_algopascal/C/biodata_procedure.c
//procedure biodata (void)

#include <stdio.h>

void biodata()
{
	char nama[20], alamat[20];
	int umur;

	printf("Masukan Nama Anda: ");
	gets(nama);
	printf("Masukan Alamat Anda: ");
	gets(alamat);
	printf("Masukan Umur: ");
	scanf("%d", &umur);

	printf("Nama Anda: %s\n", nama);
	printf("Alamat: %s\n", alamat);
	printf("Umur: %d", umur);
}

int main()
{
	biodata();
	return 0;
}